packages/contracts/contracts/TellerV2.sol

```diff
@@ -465,11 +494,8 @@ contract TellerV2 is
         Bid storage bid = bids[_bidId];
 
         address sender = _msgSenderForMarket(bid.marketplaceId);
-        (bool isVerified, ) = marketRegistry.isVerifiedLender(
```

Comment:
are we sure that it is OK to remove this ?  It seems like now anyone would be able to accept bids now -- even if not verified by the market for that market

Reply:
nevermind i see that it is handled inside of lender manager now